docs/basics.md

```diff
@@ -236,9 +236,30 @@ Here is an example of backends and servers definition:
 - `backend2` will forward the traffic to two servers: `http://172.17.0.4:80"` with weight `1` and `http://172.17.0.5:80` with weight `2` using `drr` load-balancing strategy.
 - a circuit breaker is added on `backend1` using the expression `NetworkErrorRatio() > 0.5`: watch error ratio over 10 second sliding window
 
-# Launch
+# Configuration
+
+Træfɪk's configuration has two parts: 
+
+- The [static Træfɪk configuration](/basics#static-trfk-configuration) which is loaded only at the begining. 
+- The [dynamic Træfɪk configuration](/basics#dynamic-trfk-configuration) which can be hot-reloaded (no need to restart the process).
```

Comment:
I think that you are both right :D
>Maybe we should say the the dynamic configuration is where backend are defined (and potentially watched). wdyt ?

He meant frontends, backends and servers

>Mmm, we setup the connection to configuration backends in the static configuration, right ?
The dynamic configuration is about route rules provided by configuration backends

He meant configuration backends ;)